financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentrysuccess/ManualEntrySuccessScreen.kt

```diff
@@ -18,12 +18,9 @@ internal fun ManualEntrySuccessScreen(
     val viewModel: ManualEntrySuccessViewModel = mavericksViewModel(argsFactory = { backStackEntry.arguments })
     val state by viewModel.collectAsState()
     BackHandler(true) {}
-    state.payload()?.let { payload ->
-        SuccessContent(
-            completeSessionAsync = state.completeSession,
-            payload = payload,
-            onDoneClick = viewModel::onSubmit,
-            onCloseClick = { parentViewModel.onCloseNoConfirmationClick(Pane.MANUAL_ENTRY_SUCCESS) }
-        )
-    }
+    SuccessContent(
+        completeSessionAsync = state.completeSession,
+        payloadAsync = state.payload,
+        onDoneClick = viewModel::onSubmit
+    ) { parentViewModel.onCloseNoConfirmationClick(FinancialConnectionsSessionManifest.Pane.MANUAL_ENTRY_SUCCESS) }
```

Comment:
Nit: Can we use a named parameter for this? I’m good with using trailing lambdas when their purpose is fully clear, but that’s not the case here.

Reply:
https://github.com/stripe/stripe-android/pull/8017/commits/5a5f9c3286fd99a6ac67b2db689c3e65de0843ed